### DISCUSSION TITLE
Wifi to Wifi "WISP" router

### DISCUSSION THREAD

**Discussion by aaronk6387** [RaspAP/raspap-webgui#1008] [Q&A]
If I add an USB wifi adapter to you RaspAP, can the WAN side be wireless and the LAN side be the internal wifi. Several hotels will allow 2 devices per room to log in but if i connect them all to the router it looks like one device and I can use my own private network security.
I looked and I could not find a good answer to if this works currently or not.
I have a GL-AR150 wisp capable router but it is slow. I would buy a better model from the same company but i would rather use RaspAP and have docker and everything else

**Comment by billz** [comment]
Wisp support has been [requested previously](https://github.com/RaspAP/raspap-webgui/issues/611) but other features have taken priority over it. The main reason is, frankly, most travel-type pocket routers have better wireless hardware than the Pi. We've focused on the Pi's hardware advantages and implemented features like OpenVPN / WireGuard, [network device support](https://docs.raspap.com/net-devices/) and so on.

**Reply by rbender** [reply]
What is the difference between what the original poster is asking for an what is described in the “Can I use wlan0 and wlan1 rather than eth0 for my AP?” section of the FAQ? I’m in a similar situation where I’m currently staying in a hotel and want my Pi4 to create a private wifi network for all my devices (Roku, Nintendo Switch, etc) while giving them internet access via the hotel’s public wifi (using a wifi usb dongle).
